packages/admin-ui/src/Radio/RadioItem.ts

```diff
@@ -0,0 +1,41 @@
+import { RadioItemDto } from "./RadioItemDto";
+import { generateId } from "~/utils";
+
+export class RadioItem {
+    private readonly _id: string;
+    private readonly _label: any;
+    private readonly _value: string;
+    private readonly _disabled: boolean;
+
+    protected constructor(data: { id: string; label: any; value: string; disabled: boolean }) {
+        this._id = data.id;
+        this._label = data.label;
+        this._value = data.value;
+        this._disabled = data.disabled;
+    }
+
+    static create(data: RadioItemDto) {
+        return new RadioItem({
+            id: generateId(data.id),
+            label: data.label,
+            value: data.value,
+            disabled: data.disabled ?? false
+        });
+    }
+
+    get id() {
```

Comment:
Although it might seem excessive, having a `RadioItem` model is very beneficial, and the `id` getter serves as a good example. 

While an `id` is not required when defining a radio item, it's crucial for consistency in various situations. Ensuring that each `RadioItem` has a valid `id` helps us avoid issues when interacting with or displaying items.

If an `id` isn't provided externally, the model can generate one internally, keeping everything uniform across the application.